openupgrade_scripts/scripts/website/15.0.1.0/post-migration.py

```diff
@@ -1,6 +1,49 @@
+import re
+
 from openupgradelib import openupgrade
 
 
+def extract_footer_copyright_company_name(env):
+    """Replace Copyright content in the new v15 template so as not to lose
+    content from previous versions if it has been customised."""
+    footer_copyright_company_name_view = env["ir.ui.view"].search(
```

Comment:
Why prepend all variables with `footer_copyright_company_name...` when the method is already having that name and the scope of the variables is local? Simplify them in this case: `main_footer_view`, `main_footer_arch`, etc...

Reply:
thanks!